src/main/java/eu/miaplatform/customplugin/springboot/controllers/HelloController.java

```diff
@@ -34,21 +36,16 @@
   @ApiOperation(value = "Say hello")
   @ResponseBody
   public Hello sayHello(@ApiIgnore @ModelAttribute(CP_REQUEST) CPRequest cpRequest) {
+    String userId = cpRequest.getUserId();
 
-    cpRequest.getHeadersPropagator().getHeaders().forEach(header ->
-      logger.info("headerName: " + header.getName() + " - headerValue: " + header.getValue())
-    );
-    logger.info("Hello world!");
-    return new Hello("Hello world!");
-  }
+    List<CustomPluginHeader> list = cpRequest.getHeadersPropagator().getHeaders();
+    list.forEach(customPluginHeader -> logger.info("HEADER "+customPluginHeader.getName() + " VALUE " + customPluginHeader.getValue()));
```

Comment:
Why we log all headers?

Reply:
It was for debugging purposes, I'm removing it